plugins/utils/better-escape.nix

```
@@ -0,0 +1,73 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.better-escape;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.better-escape =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "better-escape";
+
+      package = helpers.mkPackageOption "better-escape" pkgs.vimPlugins.better-escape-nvim;
```

```suggestion
      enable = mkEnableOption "better-escape.nvim";

      package = helpers.mkPackageOption "better-escape.nvim" pkgs.vimPlugins.better-escape-nvim;
```

Here, we put the name of the real plugin.